fdbclient/VersionVector.h

```diff
@@ -69,6 +73,46 @@ struct VersionVector {
 		maxVersion = invalidVersion;
 	}
 
+	void getDelta(Version version, VersionVector& delta) const {
+		ASSERT(version <= maxVersion);
+
+		delta.clear();
+
+		if (version == maxVersion) {
+			return; // rerurn an invalid version vector
+		}
+
+		std::map<Version, std::set<Tag>> versionMap;
+		for (auto& iter : versions) {
```

Comment:
Nit: you can write more readable code like:
```
for (const auto& [tag, version] : versions) {
```

Reply:
Ok, agreed.

Reply:
Done.